cmake/define-ia2-wrapper.cmake

```diff
@@ -40,6 +63,9 @@ function(define_ia2_wrapper)
         OUTPUT ${CALL_GATE_SRC} ${CALL_GATE_HDR} ${LD_ARGS_FILES}
         COMMAND ia2-rewriter
             --output-prefix=${CMAKE_CURRENT_BINARY_DIR}/${TEST_NAME}_call_gates
+            --extra-arg=-isystem "--extra-arg=${CLANG_HEADERS_INCLUDE}"
+            --extra-arg=-isystem "--extra-arg=${CLANG_HEADERS_INCLUDE_FIXED}"
```

Comment:
I think adding `-isystem` as a `target_compile_option` is preferable to this, though it depends on why we have to use clang headers in the first place.

Reply:
We don't need it for the actual compiler, just the parsing with clang part.